.github/workflows/golangci-lint.yml

```diff
@@ -15,7 +15,7 @@ permissions:
   # pull-requests: read
 jobs:
   golangci:
-    name: lint
+    name: Lint GoLang
```

Comment:
Is this the right capitalization? (I think if we're going formal, it's just "Go", we on the team add "lang" colloquially)

Reply:
Oh, interesting. I used that because the file and original name key both used it. @Laren-AWS ?

Reply:
Yep, the official name is just "Go" but a lot of people make it "Golang" for search queries and so forth. For this purpose I don't think it matters, but if you want to be strict about it, "Lint Go" is more correct.

Reply:
"GoLang" is much more searchable than "Go," so I'm going to keep it this way unless we can find a strong reason to change it.